src/modules/menu/MenuModule.vue

```diff
@@ -119,70 +124,107 @@ export default {
 @import 'src/scss/media-query.mixin';
 @import 'src/scss/variables';
 
-.header {
-    position: fixed;
-    top: 0;
-    left: 0;
-    height: $header-height;
-    transition: height 0.3s;
-    width: 100%;
-    background: rgba(255, 255, 255, 0.9);
-    box-shadow: 6px 6px 12px rgb(0 0 0 / 18%);
-    // so that the menu is above the map overlay
-    z-index: $zindex-overlay-default + 2;
-    .header-content {
-        transition: height 0.3s;
-        height: $header-height;
-    }
-}
-.menu-tray {
+$animation-time: 0.5s;
+
+.menu {
     position: absolute;
     z-index: $zindex-menu;
-    top: $header-height;
+    top: 0;
     left: 0;
     width: 100%;
-    max-width: 100%;
-    max-height: calc(100% - #{$header-height});
-    overflow-y: auto;
-    transition: 0.3s;
-    &.desktop-mode {
-        .menu-tray-content {
-            transition: opacity 0.3s;
-        }
-        max-width: 22rem;
+    height: 100%;
+    // so that the user can click through this element (and we don't block interaction with the map)
+    pointer-events: none;
+    & > * {
+        // re-activate interaction with all children of the menu
+        pointer-events: all;
```

Comment:
This is a bit hacky. But I don't have anything better on hand.
I'll create a PR when/if I have a better idea.

Reply:
yeah, I wanted to find a way to let event go through a div that would take the entire screen... Maybe there's a better way of placing elements in this div so that it doesn't take so much space

Reply:
Yeah. I'm sure there's a way to get the sizing we want without the container. But I'm drawing a blank at the moment 🙈